flutter/news/.gitignore

```diff
@@ -12,6 +12,7 @@
 .svn/
 *.mocks.dart
 *.g.dart
+*.freezed.dart
```

Comment:
Apparently on med/large project generating these files can take > 5 minutes. Something to keep in mind as it effects the developer experience and the times of execution on the CI

Reply:
Will see, thanks for the info, for now we'll keep generation on the repository so when we deal with this problem we'll figure out the best way of solving that